app/views/content/international-qualified-teacher-status/_after-accordion.md

```diff
@@ -1,15 +1,23 @@
 ## Further information
 
-Learn more about [iQTS policy](https://www.gov.uk/government/publications/international-qualified-teacher-status-iqts/introducing-the-international-qualified-teacher-status-iqts-pilot) and [Teachers’ Standards](https://www.gov.uk/government/publications/international-qualified-teacher-status-teachers-standards). 
+Learn more about [iQTS policy](https://www.gov.uk/government/publications/international-qualified-teacher-status-iqts/introducing-the-international-qualified-teacher-status-iqts-pilot) and [iQTS Teachers’ Standards](https://www.gov.uk/government/publications/international-qualified-teacher-status-teachers-standards).
 
 Learn more about [other routes to qualified teacher status (QTS) and teaching in England](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk#apply-to-the-teaching-regulation-agency-tra).
 
-[Come to England to teach if you're a teacher from outside the UK](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk) gives information and advice to international teachers who’d like to teach in England.
+[Find more help and guidance if you'd like to teach in England.](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk)
 
-If you already live in the UK, you should consider [applying for teacher training leading to QTS in England](/steps-to-become-a-teacher). 
+If you already live in the UK, you should consider [applying for teacher training leading to QTS in England](/steps-to-become-a-teacher).
 
-This guidance relates to teaching in England. Different rules apply to teaching in [Wales](https://www.ewc.wales/site/index.php/en/registration/practitioners-trained-outside-wales.html), [Scotland](https://teachinscotland.scot/become-a-teacher/qualified-outside-scotland/) and [Northern Ireland](https://gtcni.org.uk/registration/getting-registered).
+This guidance relates to teaching in England. Different rules  apply to [teaching in Wales,](https://www.ewc.wales/site/index.php/en/registration/practitioners-trained-outside-wales.html) [teaching in Scotland](https://teachinscotland.scot/become-a-teacher/qualified-outside-scotland/) and [teaching in Northern Ireland](https://gtcni.org.uk/registration/getting-registered).
```

Comment:
@valentine-carter agree this looks slightly clunky - maybe better as a bulleted list. But let's leave as is given that this page is being moved into the new section